go/client/cmd_simplefs_sync_enable.go

```diff
@@ -34,6 +36,23 @@ func NewCmdSimpleFSSyncEnable(
 	}
 }
 
+func toTlfPath(p keybase1.Path) (keybase1.Path, error) {
+	split := strings.SplitN(p.String(), "/", 4)
+	if len(split) < 4 {
+		return p, nil
+	}
```

Comment:
From [this](https://github.com/keybase/kbfs/blob/f7c9b5de11e410df6d1f4963d1c74177840b15d8/simplefs/simplefs.go#L184) it seems `keybase1.Path` can either start with a `/`, or not. So perhaps make it compatible with both to be consistent?